packages/fcl-wc/src/utils.js

```diff
@@ -0,0 +1,23 @@
+export function isAndroid() {
+  return (
+    typeof navigator !== "undefined" && /android/i.test(navigator.userAgent)
+  )
+}
+
+export function isSmallIOS() {
+  return (
+    typeof navigator !== "undefined" && /iPhone|iPod/.test(navigator.userAgent)
+  )
+}
+
+export function isLargeIOS() {
+  return typeof navigator !== "undefined" && /iPad/.test(navigator.userAgent)
+}
+
+export function isIOS() {
+  return isSmallIOS() || isLargeIOS()
+}
+
+export function isMobile() {
+  return isAndroid() || isSmallIOS()
```

Comment:
is excluding iPad intentional?

Reply:
covered by isSmallIOS no?

Reply:
According to Chrome a user agent for iPad looks something like the following "'Mozilla/5.0 (iPad; CPU OS 13_3 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/87.0.4280.77 Mobile/15E148 Safari/604.1'" so I don't think that regex would cover it.

So probably need something like:
```suggestion
  return isAndroid() || isIOS()
```